RSS Reader/RSS Reader/View Controllers/Topics list/View /Topic cell/RSSRTopicTableViewCell.m

```diff
@@ -0,0 +1,149 @@
+//
+//  RSSRTopicTableViewCell.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/17/20.
+//
+
+#import "RSSRTopicTableViewCell.h"
+#import "UIColor+RSSRColor.h"
+#import "NSDate+StringConversion.h"
+
+static NSString * const kTopicDateFormat = @"MMM d, yyyy HH:mm";
+static NSInteger const kCellViewCornerRadius = 10;
+static NSString * const kShowMore = @"Show More";
+static NSString * const kShowLess = @"Show Less";
+
+@interface RSSRTopicTableViewCell ()
+
+@property (retain, nonatomic) IBOutlet UILabel *title;
+@property (retain, nonatomic) IBOutlet UILabel *pubDate;
+@property (retain, nonatomic) UILabel *summary;
+@property (retain, nonatomic) IBOutlet UIView *cellView;
+@property (retain, nonatomic) UIButton *annotationButton;
+
+@property (nonatomic, retain) id<RSSRTopicItemProtocol> topic;
+@property (assign, nonatomic) id<RSSRTopicCellDelegate> delegate;
+
+@end
+
+@implementation RSSRTopicTableViewCell
+
+
+- (UIButton *)annotationButton {
```

Comment:
Great. You can improve it even further. If you inherit from UIButtom, you shall be able to create a convenience initializer that accepts block to handle interaction. After that this block is stored inside the button and is triggered through target action mechanism when needed. That will allow you to keep all the bottom configuration in the same place.

Reply:
Done